src/cli/serve/integration_tests/__fixtures__/reload_logging_config/kibana.test.yml

```diff
@@ -1,8 +1,6 @@
 server:
   autoListen: false
   port: 8274
-logging:
```

Comment:
Maybe we should remove the whole `legacy logging` suit including this file? https://github.com/elastic/kibana/blob/c09776dc76cf8e7c4efa1a1dd5bac2a635d66193/src/cli/serve/integration_tests/reload_logging_config.test.ts#L99